x-pack/plugins/apm/server/lib/errors/get_error_count.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ESFilter } from 'elasticsearch';
+import { oc } from 'ts-optchain';
+import { APMError } from 'x-pack/plugins/apm/typings/es_schemas/Error';
+import {
+  PROCESSOR_EVENT,
+  SERVICE_NAME,
+  TRANSACTION_ID,
+  TRANSACTION_SAMPLED
+} from '../../../common/constants';
+import { Setup } from '../helpers/setup_request';
+
+export async function getErrorCount({
+  serviceName,
+  transactionId,
+  setup
+}: {
+  serviceName: string;
+  transactionId: string;
```

Comment:
Pretty sure `trace.id` is also required here.
`transaction.id` is only unique within a trace, so (theoretically) there could be several errors with the same `transaction.id` but in different traces.